### PR TITLE
Feature: JSON flatten

### PR DESCRIPTION
**Pull request by jedrzejSmok**
function  flattening nested JSON object and array

### CODE REVIEW

File: src/main/java/pl/poznan/put/jsontools/rest/JsonToolsController.java [hunk 1]
```diff
@@ -61,6 +61,10 @@ public String get(@Valid @RequestBody JsonToolsFullRequest request) {
                         _logger.debug("Format transform added");
                         transform = new JsonTransformFormat(baseTransform);
                         baseTransform = transform;
+                    case "flatten":
+                        _logger.debug("Flatten transform added");
+                        transform = new JsonTransformFlatten(baseTransform);
+                        baseTransform = transform;
```
Comment (codeblessing) [inline src/main/java/pl/poznan/put/jsontools/rest/JsonToolsController.java:67]: Add `break;` here.

File: src/test/java/pl/poznan/put/jsontools/logic/JsonTransformRetainAttributesTest.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public void testEmptyData() {
     }
 
     @Test
-    public void testRetainAttributes(){
+    public void testRetainAttributes1(){
```
Comment (codeblessing) [inline src/test/java/pl/poznan/put/jsontools/logic/JsonTransformRetainAttributesTest.java:30]: Why this change?

**Review by codeblessing — APPROVED** [review]

**Review by pawelstrzelczyk — CHANGES_REQUESTED** [review]
all good, please apply ctrl+alt+l to format the code

**Review by pawelstrzelczyk — APPROVED** [review]
lovely

**Pull request merged**